pkgs/games/stockfish/default.nix

```diff
@@ -11,29 +11,38 @@ let
            if stdenv.isAarch64 then "armv8" else
            "unknown";
 
-    nnueFile = "nn-5af11540bbfe.nnue";
-    nnue = fetchurl {
-      name = nnueFile;
-      url = "https://tests.stockfishchess.org/api/nn/${nnueFile}";
-      sha256 = "sha256-WvEVQLv+/LVOOMXdAAyrS0ad+nWZodVb5dJyLCCokps=";
+    # These files can be found in src/evaluate.h
+    nnueBigFile = "nn-b1a57edbea57.nnue";
+    nnueBig = fetchurl {
+      name = nnueBigFile;
+      url = "https://tests.stockfishchess.org/api/nn/${nnueBigFile}";
+      sha256 = "sha256-saV+2+pXTKi4jWg3RzhFeRvrU9iF+H+G1czdVln787I=";
+    };
+    nnueSmallFile = "nn-baff1ede1f90.nnue";
+    nnueSmall = fetchurl {
+      name = nnueSmallFile;
+      url = "https://tests.stockfishchess.org/api/nn/${nnueSmallFile}";
+      sha256 = "sha256-uv8e3h+Qwd0bT3cvHv8phIghgB6BhjRdp/DrQSG9b2M=";
     };
 in
 
 stdenv.mkDerivation rec {
   pname = "stockfish";
-  version = "16";
+  version = "16.1";
 
   src = fetchFromGitHub {
     owner = "official-stockfish";
     repo = "Stockfish";
     rev = "sf_${version}";
-    sha256 = "sha256-ASy2vIP94lnSKgxixK1GoC84yAysaJpxeyuggV4MrP4=";
+    sha256 = "sha256-xTtjfJgEHF0SQT9Fw/9RLZA0Quh00jrIbihr7IYCm2U=";
   };
 
   postUnpack = ''
     sourceRoot+=/src
-    echo ${nnue}
-    cp "${nnue}" "$sourceRoot/${nnueFile}"
+    echo ${nnueBig}
```

Comment:
Is there a technical reason to `echo`?

Reply:
Apparently not, I tried removing it and the package still builds.

The `echo` was [already there](https://github.com/NixOS/nixpkgs/pull/333634/files/b8f7c30f68e5bb6b80c1be10d2934b621ead8209..4cfb5179495a6d2484e62666cb7604b2f071d52e#diff-92bcc9bd77c656dfdd19309d9131d3deff552008583b4504eb4636ac53be54a1L35) and it felt out of scope to remove it.
But I can do it here if you like.

Reply:
I can't think of a reason for keeping it. If the package builds and runs fine, I'd vote to remove it :+1:

Reply:
done!